src/main/java/net/openhft/chronicle/wire/AbstractTimestampLongConverter.java

```diff
@@ -41,13 +42,18 @@
  * <p>
  * Parsing of ISO dates with or without timestamps is supported. When an ISO date
  * is read with no timezone, it is assumed to be in the converter's zone.
- *
  */
 public abstract class AbstractTimestampLongConverter implements LongConverter {
     public static final ZoneId UTC = ZoneId.of("UTC");
     public static final String TIMESTAMP_LONG_CONVERTERS_ZONE_ID_SYSTEM_PROPERTY = "timestampLongConverters.zoneId";
+    /**
+     * To be removed in x.25, UTC dates will always be written with a Z suffix then
```

Comment:
Lets discuss with broader team first, especially as x.25 is not far away. Maybe make x.26 for now?